larsim/PhotonPropagation/SemiAnalyticalModel.cxx

```diff
@@ -99,37 +90,37 @@ void SemiAnalyticalModel::Initialization()
     x_v.push_back(elem.first);
     y_v.push_back(elem.second);
   }
-  fL_abs_vuv = std::round(interpolate(x_v, y_v, 9.7, false)); // 9.7 eV: peak of VUV emission spectrum   // TO DO UNHARDCODE FOR XENON 
+  fvuv_absorption_lenght = std::round(interpolate(x_v, y_v, 9.7, false)); // 9.7 eV: peak of VUV emission spectrum   // TO DO UNHARDCODE FOR XENON
```

Comment:
Suggest re-spelling `lenght` as `length` here and below.

Reply:
Ups! fixed now